data/credentials/credential-3.json

```diff
@@ -14,7 +14,7 @@
       "linkRelationship": "OrganizationPresentationEndpoint"
     }
   ],
-  "issuanceDate": "2016-12-31T23:59:60Z",
```

Comment:
60 is an invalid seconds value

Reply:
; ) 

These are issues that the normative requirements of the next spec will need to address.

Reply:
I would welcome some nastier example cases, for us to address.